src/main/java/com/google/jenkins/plugins/computeengine/AutofilledNetworkConfiguration.java

```diff
@@ -118,15 +110,12 @@ public ListBoxModel doFillSubnetworkItems(
         String message = "Error retrieving subnetworks";
         LOGGER.log(Level.SEVERE, message, e);
         items.clear();
-        items.add(message);
+        items.add(new ListBoxModel.Option(message, "", true));
         return items;
       }
     }
 
     public FormValidation doCheckSubnetwork(@QueryParameter String value) {
```

Comment:
Probably outside the scope of this PR, but something worth considering is do we want to check the specified subnet name against the API? More salient for the CasC use case.

Reply:
Even in the CasC use case, the subnetwork selected would need to be one of the subnetworks provided by doFillSubnetworkItems, so this would cover those failures. But I think that would be useful if more information is desired in the form validation message, because right now it's just either it failed or it didn't and it's not clear whether that was because of missing configuration or a problem with the API request.